src/translations.js

```diff
@@ -92,20 +94,75 @@ module.exports = class Translation {
   spawnTranslations () {
     let self = this;
     let alpha = moment();
-
-    console.info(`Spawning Translations started for ${self.config._LOCALES}`);
-
-    Promise.map(self.config._LOCALES, (translationLocale) => {
-      return self.downloadTranslationData(translationLocale).then( (transations) => {
-        self.updateCache(translationLocale, transations);
-        console.info(`Spawning Translations finished for ${translationLocale}`);
+    if(process.env.ORGANISATION=="CH"){
+      var options = {
+        url: this.config._LOKALISE_BASE_URL+this.config._CEMBRA_PROJECT_ID+"/files/download",
+        headers: {
+            "x-api-token": this.config._LOKALISE_API_TOKEN
+        },
+        json: {
+            "format": "json",
+            "original_filenames": false
+        },
+        gzip: true
+      }
+      request.post(options, function (err, data) {
+          if (err) {
+              return err;
+          }
+          else if (data.statusCode == 200) {
+              if (data.body.bundle_url) {
+                var reqOpts = {
+                  method: "GET",
+                  uri: data.body.bundle_url,
+                  json: true,
+                  encoding: null
+              }
+                request(reqOpts, function (err, res, body) {
+                  if (err) {
+                    return err;
+                  }
+                  var zip = new AdmZip(body);
+                  var zipEntries = zip.getEntries();
+                  async.each(zipEntries, function (zipContent, callback) {
+                      if(!zipContent.isDirectory){
+                        if(zipContent.name!=="en.json"){
+                          var locale=zipContent.name.split('_')[0];
+                          if(locale=="de"){
```

Comment:
this is confusing does locale de means translations are in german

Reply:
Yes. For German the lokalise downloads a file with name de_CH.json   so i am replacing it with CH while storing in cache

Reply:
makes sense